pkg/operator/workloadcontroller/workload_controller_openshiftapiserver_v311_00.go

```diff
@@ -44,20 +44,6 @@ func syncOpenShiftAPIServer_v311_00_to_latest(c OpenShiftAPIServerOperator, orig
 	errors := []error{}
 	operatorConfig := originalOperatorConfig.DeepCopy()
 
-	directResourceResults := resourceapply.ApplyDirectly(c.kubeClient, c.eventRecorder, v311_00_assets.Asset,
```

Comment:
@deads2k so what's your recommendation? To embed it into `workload` controller?

Isn't that a bit weird? A `daemonset` in `workload` controller depends on `statics resources` created by an external process without receiving any feedback

Reply:
> @deads2k so what's your recommendation? To embed it into `workload` controller?
> 
> Isn't that a bit weird? A `daemonset` in `workload` controller depends on `statics resources` created by an external process without receiving any feedback

The workload controller already depends on many loops like resource sync and cert rotation that are not embedded.  It expresses these dependencies in its resource.